web3.js/test/transaction.test.ts

```diff
@@ -112,6 +114,28 @@ describe('Transaction', () => {
     });
   });
 
+  it('getEstimatedFee', async () => {
+    const connection = new Connection('https://api.testnet.solana.com');
```

Comment:
Ever since we switched from Travis CI to GitHub CI, this test has been timing out on many PRs. It's surely because we're trying to talk to testnet here, right?

Can we change this into a local validator test instead? One that runs when `TEST_LIVE` is `true` (see other tests that do this).

Reply:
cc/ @marcnjaramillo

Reply:
#24730

Reply:
I commented on the issue you created. Happy to work on this with you!